src/boot/ksdk1.1.0/boot.c

```diff
@@ -3892,7 +3976,62 @@ repeatRegisterReadForDeviceAndAddress(WarpSensorDevice warpSensorDevice, uint8_t
 
 			break;
 		}
+case kWarpSensorBNO055:

```

Comment:
Case is misaligned. Needs two tabs of prefix to come in alignment.

```suggestion
		case kWarpSensorBNO055:
```